### PR TITLE
Add default values for custom force touch mode

### PR DESCRIPTION
**Pull request by tarik02**

### CODE REVIEW

File: VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist [hunk 1]
```diff
@@ -214,6 +214,12 @@
 					<integer>1</integer>
 					<key>ForceTouchPressureThreshold</key>
 					<integer>100</integer>
+					<key>ForceTouchCustomDownThreshold</key>
+					<integer>90</integer>
+					<key>ForceTouchCustomUpThreshold</key>
+					<integer>90</integer>
```
Comment (usr-sse2) [inline VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist]: Should be 20? With both values at 90 there's division by zero.

**Comment by usr-sse2** [comment]
It works, thanks! One more question: I can't understand why the lower limit is called `ForceTouchCustomUpThreshold` and the upper limit is called `ForceTouchCustomDownThreshold`.

**Comment by tarik02** [comment]
@usr-sse2 because when we reach `ForceTouchCustomDownThreshold`, the the touch is considered as full force touch down, and when we reach `ForceTouchCustomUpThreshold`, the touch is considered as touch up.

File: VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist (hunk 1 above)
Comment (tarik02) [reply VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist]: Yes, it should

**Comment by usr-sse2** [comment]
Merged manually because GitHub's merge button didn't work.

**Pull request closed without merging**